domains/prozt.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "github-username",
+    "email": "prozt709@gmail.com",
+  },
+  "record": {
+    "CNAME": "https://www.proz.tk/prozt709/"
```

Comment:
either change the 'CNAME' key to 'URL' or make the 'CNAME' key value valid of some sort.